model/plugin_status.go

```diff
@@ -10,24 +10,22 @@ import (
 
 const (
 	PluginStateNotRunning          = 0
-	PluginStateStarting            = 1
+	PluginStateStarting            = 1 // unused by server
```

Comment:
Can you elaborate? The original intent was to capture issues with the plugin starting slowly, or in the failed case below, to stop running too often. Is that just all simplified? If these are truly unused, is it time to remove them altogether?

Reply:
Ya, they are simplified. I left them in because they are natural states of the plugin that we might use in the future. Plus the client sets that state, for example when you enable a plugin on the client.